code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -401,9 +401,12 @@
 				M.emote("sneeze")
 			if(4)
 				if(prob(75))
-					to_chat(M, "You scratch at an itch.")
-					M.adjustBruteLoss(2*REM, 0)
-					. = 1
+					if(M.mobility_flags & MOBILITY_USE)
+						to_chat(M, "You scratch yourself violently.")
+						M.take_bodypart_damage(2 * REM)
+						. = 1
```

Comment:
here too

Reply:
Got carried away by every other reagent in the game doing this. Fixed now.